### PR TITLE
Fix notice "Undefined index: account-type"

### PR DESCRIPTION
**Pull request by annando**
See here: https://github.com/friendica/friendica/issues/10169#issuecomment-873660616

Possibly we should apply this fix to the stable version as well.

### CODE REVIEW

**Comment by MrPetovan** [comment]
Another unexpected continuous integration choke on the translation file even though I can't see any non-comment changes.

**Comment by annando** [comment]
The translation stuff had been caused by the release itself. That was the reason why my other PR contained the translation file.

**Review by MrPetovan — APPROVED** [review]

File: src/Model/User.php [hunk 1]
```diff
@@ -146,6 +146,15 @@ public static function getSystemAccount()
...
 		$system['nickname'] = $system['nick'];
+		$system['page-flags'] = User::PAGE_FLAGS_SOAPBOX;
+		$system['account-type'] = $system['contact-type'];
+		$system['guid'] = '';
+		$system['nickname'] = $system['nick'];
+		$system['pubkey'] = $system['pubkey'];
+		$system['locality'] = '';
+		$system['region'] = '';
+		$system['country-name'] = '';
+		$system['net-publish'] = false;
```
Comment (AlfredSK) [inline src/Model/User.php:157]: `$system['nickname'] = $system['nick'];`

Umm. This line exists two times.

Comment (MrPetovan) [reply src/Model/User.php:157]: Better safe than sorry!

**Pull request merged**